user_guides/saaj/src/main/jbake/content/toc.adoc

```diff
@@ -63,9 +63,9 @@ link:install.html#GBFTP[3 Installation]
 link:config.html#GBFVV[4 Setup and Configuration]
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-** link:config.html#GHGDH[4.1 Configuring Your Environment to Run the TCK With a Compatible Implementation]
-** link:config.html#GHGDH[4.2 Configuring Your Environment to Run the TCK With Your Concurreny Implementation]
-** link:config.html#GHGDH[4.3 Publishing the Test Applications]
+** link:config.html#GBFVU[4.1 Configuring Your Environment to Run the TCK With a Compatible Implementation]
+** link:config.html#GCLHU[4.2 Configuring Your Environment to Run the TCK With Your Concurreny Implementation]
+** link:config.html#GHGDG[4.3 Publishing the Test Applications]
```

Comment:
in general - `toc.adoc` files are being created by the build, so ideally - existing ones should be deleted/removed from the repo, `clean` goal updated to delete this file during `mv clean` and then added to `.gitignore` to not accidentally commit these files again

Reply:
Thanks for extra explanation.